corehq/warehouse/states.py

```diff
@@ -0,0 +1,34 @@
+'''
+When a fact table is in the "needs_updating" state, it will be
+selected for processing during the task that determines which
+fact tables need updating.
+'''
+FACT_TABLE_NEEDS_UPDATING = 'needs_updating'
+
+'''
+When a fact table has reached the batch dumped state, it will
```

Comment:
nitpicky: Can you change this to start with "When a fact table is in"? Or change the rest of the comments os that these are in the same format?